.jenkins/continuous.groovy

```diff
@@ -412,25 +412,30 @@ pipeline {
                         sh 'rm -rf build && mkdir -p build'
                         dir('build') {
                             sh '''
+                                . /opt/intel/oneapi/setvars.sh --include-intel-llvm && \
                                 cmake \
                                     -D CMAKE_INSTALL_PREFIX=$ARBORX_DIR \
                                     -D CMAKE_BUILD_TYPE=Release \
-                                    -D CMAKE_CXX_COMPILER=clang++ \
+                                    -D CMAKE_CXX_COMPILER=/opt/intel/oneapi/compiler/2023.0.0/linux/bin-llvm/clang++ \
                                     -D CMAKE_CXX_EXTENSIONS=OFF \
-                                    -D CMAKE_CXX_FLAGS="-fsycl-device-code-split=per_kernel -Wpedantic -Wall -Wextra -Wno-unknown-cuda-version" \
-                                    -D CMAKE_PREFIX_PATH="$KOKKOS_DIR;$BOOST_DIR;$BENCHMARK_DIR;$ONE_DPL_DIR" \
+                                    -D CMAKE_CXX_FLAGS="-fp-model=precise -fsycl-device-code-split=per_kernel -Wpedantic -Wall -Wextra -Wno-unknown-cuda-version" \
+                                    -D CMAKE_PREFIX_PATH="$KOKKOS_DIR;$BOOST_DIR;$BENCHMARK_DIR" \
                                     -D ARBORX_ENABLE_MPI=ON \
                                     -D MPIEXEC_PREFLAGS="--allow-run-as-root" \
                                     -D MPIEXEC_MAX_NUMPROCS=4 \
                                     -D ARBORX_ENABLE_TESTS=ON \
                                     -D ARBORX_ENABLE_EXAMPLES=ON \
                                     -D ARBORX_ENABLE_BENCHMARKS=ON \
-                                    -D ARBORX_ENABLE_ONEDPL=ON \
-                                    -D ONEDPL_PAR_BACKEND=serial \
```

Comment:
We cannot use ONEDPL with that image?

Reply:
Yes, we can. We make sure that the compiler already has the appropriate includes and doesn't need to find a  `CMake` package. We would error out if `oneDPL` could not be found (since it's enabled by default).

Reply:
Is that something you intend to revisit later?

Reply:
No, this is using oneDPL.

Reply:
How so?  Presumably the macro `ARBORX_ENABLE_ONEDPL` won't be defined.

Reply:
I don't understand, `ARBORX_ENABLE_ONEDPL` is by default `ON` when `Kokkos_ENABLE_SYCL` is defined.
It seems it's easier to just be explicit and add it back.

Reply:
My bad I misread the cmake_dependent_option.  Sorry about the noise.